pkgs/by-name/ru/rutabaga_gfx/package.nix

```diff
@@ -40,8 +40,19 @@ stdenv.mkDerivation (finalAttrs: {
     })
     # Install the dylib on Darwin.
     ./darwin-install.patch
+    # Patch for libc++, drop in next update
+    # https://issuetracker.google.com/issues/392934337
+    (fetchpatch {
+      url = "https://chromium.googlesource.com/crosvm/crosvm/+/8ae3c23b2e3899de33b973fc636909f1eb3dc98c%5E%21/?format=TEXT";
+      decode = "base64 -d";
+      hash = "sha256-RL9W2nU3bVuNu23eNOyY6Ne6Lm5xz8bBvTqvyskFSPc=";
+    })
   ];
 
+  preConfigure = lib.optionalString stdenv.hostPlatform.useLLVM ''
+    export USE_CLANG=1
+  '';
```

Comment:
`env`?  There's no need to drop to shell here.

Reply:
It's to prevent this error outside `pkgsLLVM`:
```
       >   = note: /nix/store/gx5qlmsiccjv8v39l0mpikksl1pgssnw-binutils-2.43.1/bin/ld: cannot find -lc++: No such file or directory
       >           collect2: error: ld returned 1 exit status
       >
       >
       > warning: `rutabaga_gfx_ffi` (lib) generated 1 warning
       > error: could not compile `rutabaga_gfx_ffi` (lib) due to 1 previous error; 1 warning emitted
       > make: *** [Makefile:47: build] Error 101
```

Reply:
```nix
env = lib.optionalAttrs stdenv.hostPlatform.useLLVM {
  USE_CLANG = "1";
};
```